notebooks/dashboard.ipynb

```diff
@@ -447,7 +456,41 @@
     "    zooms_df = pd.DataFrame(list(chain(*zoom_df['zoom'].values.tolist())))\n",
     "    zooms_df['img_cat'] = pd.factorize(zooms_df['img_idx'])[0]\n",
     "else:\n",
-    "    zooms_df = pd.DataFrame(dict(x=[], y=[], img_idx=[], img_cat=[]))"
+    "    zooms_df = pd.DataFrame(dict(x=[], y=[], img_idx=[], img_cat=[]))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_bbox_handles(in_view_dict):\n",
```

Comment:
move this guy to `jupyanno/cornerstone.py`?

I'd also add the below as a helper in `utils.py` to make things easier to read:
```python
def getIn(a_dict, *args):
  value = a_dict.get(args[0], None):
  for arg in args[1:]:
    if value is not None:
      value = value.get(arg, None)
  return value
```

Reply:
I think it probably makes more sense to have the code in cornerstone_widget https://github.com/chestrays/cornerstone_widget/issues/8

Reply:
@betatim we actually do not want an exception because lots of images won't have any bounding boxes (healthy patients) or contain other information (like the plotly or standard image widget) and we don't want it to fail on those.